doc/rtmpwebhookauth.md

```diff
@@ -10,9 +12,9 @@ For example:
 livepeer -authWebhookUrl http://ownserver/auth
 ```
 
-For each incoming RTMP stream, the Livepeer node will make a `POST` request to the `http://ownserver/auth` endpoint, passing the URL of the RTMP request as JSON object.
+For each incoming stream, the Livepeer node will make a `POST` request to the `http://ownserver/auth` endpoint, passing the URL of the RTMP request as JSON object.
```

Comment:
"passing the URL of the RTMP request" -> "passing the URL of the request"?

Reply:
Fixed in bf686d88df7a7bc34da1baca46351a730d83fa37 , and a number of other usages of "RTMP" that were also missed.